lib/node_modules/@stdlib/stats/base/varianceyc/test/test.ndarray.js

```diff
@@ -104,6 +161,22 @@ tape( 'if provided an `N` parameter equal to `1`, the function returns a populat
 	t.end();
 });
 
+tape( 'if provided an `N` parameter equal to `1`, the function returns a population variance of `0` (accessor)', function test( t ) {
+	var x;
+	var v;
+
+	x = toAccessorArray([ 1.0, -2.0, -4.0, 5.0, 3.0 ]);
```

Comment:
```suggestion
	x = toAccessorArray( [ 1.0, -2.0, -4.0, 5.0, 3.0 ] );
```

This same change applies throughout this file. Use consistent spacing.